DFC.App.DiscoverSkillsCareers/ViewModels/Assessment/AssessmentReferencePostRequest.cs

```diff
@@ -6,7 +6,10 @@ namespace DFC.App.DiscoverSkillsCareers.ViewModels
     [ExcludeFromCodeCoverage]
     public class AssessmentReferencePostRequest
     {
-        [Required]
+        [Required(ErrorMessage = "Enter a phone number")]
+        [RegularExpression(@"^(((\+44\s?\d{4}|\(?0\d{4}\)?)\s?\d{3}\s?\d{3})|((\+44\s?\d{3}|\(?0\d{3}\)?)\s?\d{3}\s?\d{4})|((\+44\s?\d{2}|\(?0\d{2}\)?)\s?\d{4}\s?\d{4}))(\s?\#(\d{4}|\d{3}))?$",
```

Comment:
Did you get the regex from here? -> https://skillsfundingagency.atlassian.net/wiki/spaces/DFC/pages/1919091155/CUI+field+validation+rules